Upstream: yes
Source: upstream, http://cvsweb.openbsd.org/cgi-bin/cvsweb/src/lib/libssl/src/ssl/ssl.h.diff?r1=1.73&r2=1.74&f=h
Reason: compilation failures (e.g. mutt: "/usr/include/openssl/ssl.h:503:2: error: unknown type name 'uint8_t'")

--- libressl-2.1.2/include/openssl/ssl.h.orig	2014-12-13 12:06:24.675054404 +0100
+++ libressl-2.1.2/include/openssl/ssl.h	2014-12-13 12:08:16.611961114 +0100
@@ -1,4 +1,4 @@
-/* $OpenBSD: ssl.h,v 1.72 2014/11/18 05:33:43 miod Exp $ */
+/* $OpenBSD: ssl.h,v 1.73 2014/12/06 13:51:06 jsing Exp $ */
 /* Copyright (C) 1995-1998 Eric Young (eay@cryptsoft.com)
  * All rights reserved.
  *
@@ -143,6 +143,8 @@
 #ifndef HEADER_SSL_H 
 #define HEADER_SSL_H 
 
+#include <stdint.h>
+
 #include <openssl/opensslconf.h>
 #include <openssl/hmac.h>
 #include <openssl/pem.h>
